config/config.go

```diff
@@ -104,7 +104,7 @@ func processRawConfig(rawConfig RawConfig, config *Config) (*Config, error) {
 	}
 
 	for i, chain := range rawConfig.ChainConfigs {
-		if len(config.ChainConfigs) < len(rawConfig.ChainConfigs) {
+		if i >= len(config.ChainConfigs) {
```

Comment:
If we update the relayers and add a new domain and then update the shared config to avoid something breaking in between those two times this is added as a safety.

Reply:
It happened when we had Shibuya in relayers, but not yet in shared config and then stuff broke. 
If we want to test stuff on relayers before adding it to everything it is a nice to have.